pantheon/src/main/java/tech/pegasys/pantheon/controller/CliquePantheonController.java

```diff
@@ -235,6 +235,11 @@ private CliquePantheonController(
           } catch (final IOException e) {
             LOG.error("Failed to close storage provider", e);
           }
+          try {
```

Comment:
We should stop the synchronizer prior to the storage.  The shutdown process could try to write more data.